addEvent.php

```diff
@@ -328,76 +267,124 @@
-
                 <p>Responsible Party's Name </p>
                 <input type="text" id="AAA_membership_name" name="AAA_membership_name" required placeholder="Enter name">
-
                 <p> Responsible Party's Date of Birth </p>
                 <input type="date" id="AAA_membership_DOB" name="AAA_membership_DOB" <?php if ($date) echo 'value="' . $date . '"'; ?> min="<?php echo date('Y-m-d'); ?>" required>
-
                 <label for="name"> Photography </label>
                     <p> 
                     We offer your family two sessions of photography.  
                     We will do one during treatment and again after treatment has finished. 
                     There is no charge to your points for this service. 
                     </p> 
-
                 <label for="name">* Are you interested in a photography session? </label>
                     <?php 
                         echo '<ul>';
                         echo '<li><input class="radio" type="radio" name="photography" value="' . "yes" . '" required/> ' . "Yes" . '</li>';
                         echo '<li><input class="radio" type="radio" name="photography" value="' . "no" . '" required/> ' . "No" . '</li>';
                         echo '</ul>';
                     ?>
-
                 <label for="name"> Additional Services </label>
                     <p>
                     We currently offer to help with house projects and financial relief twice a year.  
                     If you are interested we will contact you when these services become available
                     </p>   
-
                 <label for="name">* House Projects </label>
                     <?php 
                         echo '<ul>';
                         echo '<li><input class="radio" type="radio" name="house_projects" value="' . "more_info" . '" required/> ' . "We would like more information when available" . '</li>';
                         echo '<li><input class="radio" type="radio" name="house_projects" value="' . "not_interested" . '" required/> ' . "We are not interested in house projects" . '</li>';
                         echo '</ul>';
                     ?>
-
                 <label for="name">* Financial Relief </label>
                     <?php 
                         echo '<ul>';
                         echo '<li><input class="radio" type="radio" name="financiel_relief" value="' . "more_info" . '" required/> ' . "We would like more information when available" . '</li>';
                         echo '<li><input class="radio" type="radio" name="financiel_relief" value="' . "not_interested" . '" required/> ' . "We are not interested in financial relief" . '</li>';
                         echo '</ul>';
                     ?>
-
                 <input type="submit" value="Submit">
+                                <!--Commented out original ODHS code-->
```

Comment:
This commit just includes reorganization of the points program form code. The commented out ODHS code was moved to the bottom of the file and extraneous whitespace lines were removed. Great work keeping the code organized so our workspace is clean and easy to use. Keeping the most relevant code at the top is a good way to do that.

Reply:
The reorganization was key to getting the form code completed. I believe the unused code may be useful for us to use as a template later on, should we keep the addEvent.php file within our workspace to reference?